mkdocs.yml

```diff
@@ -255,4 +258,4 @@ extra:
     repo: https://github.com/cozy/cozy-coclyco.git
     subdir: .
   search:
-    tokenizer: "[^a-z\u0430-\u044F\u04510-9\\-\\._]"
+    tokenizer: "[^a-z\u0430-\u044F\u04510-9\\-\\.]"
```

Comment:
This change should be in another commit and should be documented.

Reply:
Not sure where this comes from, is it because the mkdocs.yml was not up to date?

Reply:
Ah sure, you're right. I was surprised to see changes to the mkdocs.yml but I understand now : Iit's not really a problem if it's not up-to-date since it's a generated file. It could be removed from the repository, I think it's a bit misleading to have it here.

Reply:
Ah ok, so we should add it to .gitignore?

Reply:
Done with 930822b3fa4f09cebe4bb3bfe7373fbcc5aa0d67